src/helpers/actions.ts

```diff
@@ -39,18 +39,36 @@ export const clickOnLogo = async (page: Page): Promise<void> => {
   await header.click();
 };
 
+/**
+ *
+ * @param page
+ * @param label
+ * @param text
+ * @param clear
+ * @param excludeSpan
+ * @param optional
+ * @returns true if found and updated, false otherwise
```

Comment:
can you point usage of return value?

Reply:
When filling gas details pre-eip1559, the user might submit priority which would try to write a non-existing field.